### DISCUSSION TITLE
How to exclude work item types other than Epics and Features

### DISCUSSION THREAD

**Discussion by mbizkit76** [nbrown02/FlowViz#54] [Q&A] [answered]
Hi, given that I'm a newbie in using PowerBI, I wonder if there is a way to change the WorkItems Type filter in order to exclude for example Retrospectives
Thanks
Marco

**Comment by nbrown02** [comment, marked as answer]
Hey Marco,

There a couple ways you can do it. A more advanced way would be edit the query, however if you're new to Power BI I wouldn't suggest that :D The easier way is to Open PBIT file and load data in Power BI Desktop then use the filters pane - Click on View -> Filters and it will appear, you can then either multi-select/unselect for the work item types to include/exclude, like so:
![1](https://user-images.githubusercontent.com/29369962/157731229-2a7245fa-abd5-4356-a4a3-f5273e74b2c1.gif)


Or drag/drop the WorkItemType fields into this again and filter out the ones you don't need...

**Reply by mbizkit76** [reply]
Ok Thanks a lot. I couldn't figure out because I was using PowerBI webapp instead of Desktop app and this View -> Filters pan was not available

**Reply by nbrown02** [reply]
Ah yes good point, I shall edit my answer to show that :)
